helix-term/src/ui/editor.rs

```diff
@@ -697,6 +738,242 @@ impl EditorView {
         );
     }
 
+    /// Render the sticky context
+    #[allow(clippy::too_many_arguments)]
+    pub fn render_sticky_context(
+        doc: &Document,
+        view: &View,
+        surface: &mut Surface,
+        context: &Option<Vec<StickyNode>>,
+        doc_annotations: &TextAnnotations,
+        line_decoration: &mut [Box<dyn LineDecoration + '_>],
+        translated_positions: &mut [TranslatedPosition],
+        theme: &Theme,
+    ) {
+        if context.is_none() {
+            return;
+        }
+
+        let context = context.as_deref().expect("context has value");
+
+        let text = doc.text().slice(..);
+        let viewport = view.inner_area(doc);
+
+        // TODO: this probably needs it's own style, although it seems to work well even with cursorline
+        let context_style = theme.get("ui.cursorline.primary");
```

Comment:
```suggestion
        let context_style = theme.get("ui.virtual.sticky-context");
```

Or just `ui.virtual.context`